lib/mongo/events.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Mongo.Events.ServerDescriptionChangedEvent do
```

Comment:
It's not supposed to be internal-only. I will document these structs tomorrow. I can add the `GenEvent.notify` as a method to `Mongo.Events`, too.